css/index.css

```diff
@@ -0,0 +1,84 @@
+body{
+    background-color: whitesmoke;
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+    font-size: 12px;
+    
+}
+
+h1{
+    font-size: 25px;
+    font-style: oblique;
+}
+
+h2{
+    color: cornflowerblue;
+}
+
+p{
+    font-size: 12px;
+    color: black;
+}
+
+a {
+	color: #18272F;
+    position: relative;
+    text-decoration: none;
+}
+
+a::before {
+  content: '';
+  position: absolute;
+  width: 100%;
+  height: 4px;
+  border-radius: 4px;
+  background-color: #18272F;
+  bottom: 0;
```

Comment:
I really like the design of the links, the underlining is a nice addition. However, it’s currently covering the bottom part of the text, which makes it a bit hard to read. You could try moving it slightly lower by adjusting the bottom value, for example: `bottom: -5px;`. Another option is to try changing the opacity percentage to make the line more subtle. Once you've made the changes, take a look at how it looks on different letters (especially letters which could be covered more, like 'g' or 'y').

Reply:
Thank you so much for your attention Veneta! I will verify that.